test/jdk/javax/swing/JTable/KeyBoardNavigation.java

```diff
@@ -150,7 +99,7 @@ static void initTest() throws Exception {
         colorColumnRenderer.setToolTipText("Click for combo box");
         colorColumn.setCellRenderer(colorColumnRenderer);
 
-        // Set a tooltip for the header of the colors column.
+        // Set a tooltip for the header of the color's column.
         TableCellRenderer headerRenderer = colorColumn.getHeaderRenderer();
         if (headerRenderer instanceof DefaultTableCellRenderer)
             ((DefaultTableCellRenderer) headerRenderer).setToolTipText("Hi Mom!");
```

Comment:
Suggestion:
Usage of enhanced `instanceOf` avoids the casting to `DefaultTableCellRenderer` below. Use `{ }` for single if statement too.

```suggestion
        if (colorColumn.getHeaderRenderer() instanceof DefaultTableCellRenderer headerRenderer ) {
           headerRenderer.setToolTipText("Hi Mom!");
       }
```

Same can be used at L115 as well.
`int cellValue = (value instanceof Number number) ? number.intValue() : 0;`

Reply:
Yeah, updated.